### PR TITLE
Removed defaultWildcardSecret check in cert-secret.yaml as argocd doe…

### PR DESCRIPTION
**Pull request by joycegilding-forgerock**
Removed defaultWildcardSecret check in cert-secret.yaml as argocd does not handle boolean flag properly and throws an error.

To generate dev cert, set the externalCert.prefix to "dev"
Add externalCert.certPrex and externalCert.secretName parameter to namespace-init.yaml so that user can pass in developer namespace using the argo cd application.
Updated README
Issue: https://github.com/SecureBankingAccessToolkit/SecureBankingAccessToolkit/issues/294

### CODE REVIEW

File: cluster/certificate/values.yaml [hunk 1]
```diff
@@ -1,4 +1,3 @@
-defaultWildcardSecret: false
```
Comment (BohoCode) [inline cluster/certificate/values.yaml:1]: And this!

File: argo/apps/values.yaml [hunk 2]
```diff
@@ -44,6 +44,8 @@ mongodb:
 
 externalCert:
   projectId: example-project
+  secretName: sslcert
+  certPrefix: dev
```
Comment (BohoCode) [inline argo/apps/values.yaml:48]: Ah, this must be why PR wasn't working!

**Review by BohoCode — APPROVED** [review]

File: cluster/certificate/values.yaml (hunk 1 above)
Comment (jorgesanchezperez) [reply cluster/certificate/values.yaml:1]: You can use `--set-string` to set properly the true/false flag.

Comment (BohoCode) [reply cluster/certificate/values.yaml:1]: We don't need it any more

**Pull request merged**